Gemfile.lock

```diff
@@ -337,6 +337,7 @@ GEM
     mysql2 (0.4.9-x64-mingw32)
     mysql2 (0.4.9-x86-mingw32)
     nio4r (2.1.0)
+    nio4r (2.1.0-java)
```

Comment:
Please revert this!!!

Reply:
It's fine.. Bundler can never make up its mind about whether those lines should be there, but AFAICS they should.